internal/cli/run.go

```diff
@@ -56,22 +57,20 @@ func (s Service) RunSuite(ctx context.Context, cfg RunConfig) (finalErr error) {
 		}
 	}
 
-	commandArgs := make([]string, 0)
-
-	if cfg.Command != "" {
-		parsedCommand, err := shellwords.Parse(cfg.Command)
-		if err != nil {
-			return errors.Wrapf(err, "Unable to parse %q into shell arguments", cfg.Command)
-		}
-		commandArgs = append(commandArgs, parsedCommand...)
+	runCommand, err := s.makeRunCommand(ctx, cfg)
+	if err != nil {
+		return errors.Wrapf(err, "Failed to assemble run command")
 	}
 
-	if len(cfg.Args) > 0 {
-		commandArgs = append(commandArgs, cfg.Args...)
+	// Short circuit and print warning info (e.g attempting to run an empty partition)
+	if runCommand.shortCircuit {
+		s.Log.Warnf(runCommand.shortCircuitInfo)
+		os.Exit(0)
```

Comment:
Curious if folks think this is appropriate.

Reply:
I think so. What's the alternative?

Reply:
alternative would be to move `shortCircuitInfo` into error... which I think might make more sense? When is `ShortCircuit` _not_ a misconfiguration?

but also I think it's fine for now.

Reply:
sorry about the long winded serial comments! 

A question is: "should this break the build" and my opinion is: yes it should. So maybe an error is appropriate. 

what is "this" though -- too many partitions? That should, in in 90% of cases, break the build.

Reply:
Yeah, "too many partitions". I'm inclined to _not_ have it break a build, personally. I suspect it's pretty rare in practice 😅